server/src/modules/customConfig/customConfig.repository.ts

```diff
@@ -10,25 +10,24 @@ export class CustomConfigRepository {
     private readonly customRepository: Repository<CustomConfigEntity>,
   ) {}
 
-  async readSpotFee(configId: number = 0) {
+  private async getConfigById(configId: number): Promise<CustomConfigEntity> {
     const config = await this.customRepository.findOne({
       where: { config_id: configId },
     });
     if (!config) {
-      // Handle config not found error
-      return;
+      // handle config not found error
```

Comment:
This will simplify flow but changes logic: 
right now lack of config will throw an error, in initial implementation, undefined would be returned, to consider if this is right behaviour

Reply:
It's alright, we need an initial config to start anyway, configId 0 will always be there